source/time.tex

```diff
@@ -10938,7 +10939,6 @@
 };
 \end{codeblock}
 
-% FIXME: Decide how to index this. This template-id is a lie.
```

Comment:
Now you've removed this totally unrelated FIXME.

Could I ask you limit this change to just the indexing snafu? Thanks.

Reply:
This _is_ the snafu. It's a previously known one, just not one that was previously resolved...